lib/assignDisabledRanges.js

```diff
@@ -53,7 +53,41 @@ module.exports = function (root, result) {
 	};
 
 	result.stylelint.disabledRanges = disabledRanges;
-	root.walkComments(checkComment);
+
+	// Work around postcss/postcss-scss#109 by merging adjacent `//` comments
+	// into a single node before passing to `checkComment`.
+
+	/** @type {PostcssComment?} */
+	let inlineEnd;
+
+	root.walkComments((/** @type {PostcssComment} */ comment) => {
+		if (inlineEnd) {
+			// Ignore comments that were already processed by grouping with
+			if (inlineEnd === comment) inlineEnd = null;
+		} else if (comment.inline || comment.raws.inline) {
```

Comment:
Add a comment explaining why both of these need to be checked. Maybe factor this out into an `isInlineComment()` function?

Reply:
Done